control/available_plugin.go

```diff
@@ -75,6 +75,7 @@ type availablePlugin struct {
 	exec               string
 	execPath           string
 	fromPackage        bool
+	pprofPort          string
```

Comment:
Same reason, we have no reason to use it as an int. The goal is to displayed it at the end.